### DISCUSSION TITLE
mistake CPU profit

### DISCUSSION THREAD

**Discussion by lukras** [RainbowMiner/RainbowMiner#1724] [General]
![CPU](https://user-images.githubusercontent.com/30388030/133597268-92dbb8b2-ec12-4143-a216-2f1192250186.png)
Intel(R) Xeon(R) CPU           E5520  @ 2.27GHz (CPU#00)
========================================================

Miner            Fee Algorithm        Speed Diff Power   BTC/Day  ETH/Day  RUB/Day  USD/Day  Accuracy Pool   PoolFee
-----            --- ---------        ----- ---- -----   -------- -------- -------- -------- -------- ----   -------
SrbMinerMulti 0.85 % Yespower2b 331.37  H/s    -  105W 0.01265368 0.175368 41,158.0  565.544     99 % RPlant    1. %
===============================================================================

Please correct for real profit  :-)  its not 565 USD/DAY

**Comment by RainbowMiner** [comment]
Oh dear! I am sorry, I did miss this question. Tomorrow I'll have a look into that.
